var/spack/repos/builtin/packages/openmx/package.py

```diff
@@ -41,6 +49,10 @@ def edit(self, spec, prefix):
         # Move contents to source/
         # http://www.openmx-square.org/bugfixed/18June12/README.txt
         copy_tree('patch', 'source')
+        # Move extra file for 3.9 patch
+        # http://www.openmx-square.org/bugfixed/21Oct17/README.txt
+        if spec.satisfies('@3.9'):
```

Comment:
Is the way current patch is applied to 3.9 version.
Can't tell for future patches

Reply:
Thanks for the pointer. It's quite confusing to see 3.9.9 in the patch and 3.9 in the release number, but that's somehow a choice of the upstream project.

I see though that there are at least two patches for version 3.9. We might consider reworking this package in the future, if it seems sensible, to use 3.8.5 and 3.9.9 instead of 3.8 and 3.9. This would allow us to add new "patch" versions without removing the ones previously present.